astropy/visualization/wcsaxes/patches.py

```diff
@@ -81,9 +83,15 @@ class SphericalCircle(Polygon):
 
     def __init__(self, center, radius, resolution=100, vertex_unit=u.degree, **kwargs):
 
-        # Extract longitude/latitude, either from a tuple of two quantities, or
-        # a single 2-element Quantity.
-        longitude, latitude = center
+        # Extract longitude/latitude, either from a SkyCoord object, or
+        # from a tuple of two quantities or a single 2-element Quantity.
+        if isinstance(center, SkyCoord):
+            if center.frame.name == 'galactic':
+                longitude, latitude = center.l, center.b
```

Comment:
Wouldn't it be better to use `center.representation_component_names` to find out what the longitude and latitude of `center` are called instead of hard-coding the names? It would also allow the user to provide a `SkyCoord` in e.g. ecliptic coordinates, which is not possible in this implementation.